src/org/redalert1741/steamworks/SwerveDrive.java

```diff
@@ -175,6 +175,76 @@ void Swerve(double x, double y, double z, double gyro, boolean fieldOrient)
 		}
 	}
 	
+	void SwerveAbsolute(double x, double y, double z, double gyro, boolean fieldOrient)
```

Comment:
You might as well replace the original Swerve method with this implementation. That should cut down on errors copying and pasting between the two.